src/AnimatedLineGraph.tsx

```diff
@@ -215,16 +229,22 @@ export function AnimatedLineGraph({
     [isActive, setIsActive]
   )
   const positions = useDerivedValue(
-    () => [
-      0,
-      Math.min(0.15, pathEnd.current),
-      pathEnd.current,
-      pathEnd.current,
-      1,
-    ],
+    () => [0, pathEnd.current, pathEnd.current, pathEnd.current, 1],
```

Comment:
Why this change? It can happen that you pan to the start of the graph, in which case the gradient won't work correctly